lib/cc/engine/scalastyle.rb

```diff
@@ -57,22 +57,28 @@ def analysis_files
       end
 
       def scalastyle_jar
-        '/home/app/scalastyle_2.11-0.6.0-batch.jar'
+        '/home/app/scalastyle_2.11-0.8.0-batch.jar'
       end
 
       def scalastyle_config
-        @engine_config['config'] || '/home/app/scalastyle_config.xml'
+        @engine_config['config'] || '/code/scalastyle_config.xml'
       end
 
       def results_xml_file_path
         "/home/app/results.xml"
       end
 
+      def results_xml_tmp_file_path
+        "/home/app/results.tmp.xml"
+      end
+
       def exec_scalastyle_with_options
+        Dir.chdir "/code"
```

Comment:
It's a good practice (IMO) to use the block form of `Dir.chdir` so the working directory is restored after.